.github/workflows/rollback.yml

```diff
@@ -1,11 +1,287 @@
 name: Rollback
+run-name: Rollback the ${{ inputs.service }} service in the ${{ inputs.environment }} environment
 
 on:
   workflow_dispatch:
+    inputs:
+      service:
+        type: choice
+        description: The microservice which we want to rollback to the previous version.
+        options:
+          - inbound_mail
+          - api
+          - web
+          - webhook
+          - widget
+          - worker
+          - ws
+      environment:
+        type: choice
+        description: The environment where the service is located.
+        options:
+          - Development
+          - Production
+      region:
+        type: choice
+        description: The region of the environment. It's needed to know TF workspace. When you choose DEV environment this option is not needed.
```

Comment:
```suggestion
        description: Select the environment region. Required only in production.
```

Reply:
TIL that Github actions don't support [dynamic, dependent inputs yet.](https://github.com/orgs/community/discussions/11795)

Reply:
> TIL that Github actions don't support [dynamic, dependent inputs yet.](https://github.com/orgs/community/discussions/11795)

It's a very useful feature